eng/scripts/build.ps1

```diff
@@ -63,7 +63,7 @@ $root = Resolve-Path ($PSScriptRoot + "/../..")
 Set-Location $root
 $sdks = @{};
 
-foreach ($sdk in (Get-ModuleDirs 'sdk/...')) {
+foreach ($sdk in (Get-ModuleDirs 'sdk/')) {
```

Comment:
Why do we have to change this?

Reply:
I think the function is to find all module dirs in sdk subfolders. So the three dots in path are not necessary and also Get-ChildItem with three dots path not work in linux environment (ci base image is ubuntu).